locale/en-us/unexpected_error.dialog

```diff
@@ -0,0 +1 @@
+Unexpected answer
```

Comment:
Consider something more conversational, i.e.:
`Sorry, I'm not sure what you mean.` or something that makes sense contextually where this is spoken.